client/src/components/pages/Home.js

```diff
@@ -0,0 +1,73 @@
+import { React, useEffect } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { Grid } from "@material-ui/core";
+import { AccountCircle } from "@material-ui/icons";
+import { getUser } from "../../store/actions/userActions";
+import { getAllJobs } from "../../store/actions/jobPostActions";
+import { TableUsers, CreateJobButton } from "../molecules/index";
+import { ReactComponent as Banner } from "../../assets/Home/home_asset.svg";
+
+import "./styles/Home.css";
+
+const Home = (props) => {
+  useEffect(() => {
+    props.actions.getUser(props.user.uid);
+    props.actions.getAllJobs();
+  }, [props.actions, props.user]);
+
+  const firstName = props.users.user?.name?.split(" ")[0];
+
+  return (
+    <div className="home_page">
+      <Banner className="home_banner"/>
+      <Grid container spacing={2} direction="column">
+        <Grid container alignItems="center" className="home_header">
+          <Grid item className="home_header_picture">
+            {props.users.user?.profilePicture ? (
+              <img
+                src={props.users.user?.profilePicture}
+                alt="Profile pic"
+                className="profile_img"
+              />
+            ) : (
+              <AccountCircle
+                style={{ fontSize: "6em" }}
+                className="profile_image_default"
+              />
+            )}
+          </Grid>
+          <Grid item className="home_header_bio">
+            <h1 style={{ marginTop: 0 }}>Hi, {firstName}</h1>
+            <p style={{ marginBottom: 0 }}>
+              Take a look at the community's postings!
+            </p>
+          </Grid>
+        </Grid>
+
+        <Grid item style={{ paddingLeft: "15%"}}>
+            <TableUsers data={props.jobs.currentListOfJobs} />
```

Comment:
is currentListOfJobs supposed to hold jobs related to a specific user or related to all job postings in the database? bc I recall it's being used on profile page to list out the jobs associated with a user. Maybe to be safe, change this to use a new list -> 'allJobPostingsList' instead.